main.go

```diff
@@ -191,17 +191,17 @@ func initFlags(fs *pflag.FlagSet) {
 func validateFlags() error {
 	switch options.ProviderIDFormatType(options.PowerVSProviderIDFormat) {
 	case options.PowerVSProviderIDFormatV1:
-		setupLog.Info("Using v1 version of Power VS ProviderID format")
+		setupLog.Info("Power VS ProviderID format is set to v1 version, Note: its deprecated use provider-id-fmt to provider id format")
```

Comment:
with the above `MarkDeprecated` call, you may not need this explicit message anymore